objective.py

```diff
@@ -5,70 +5,86 @@
     import numpy as np
     from numpy.linalg import norm
 
-    from skglm.datafits import QuadraticGroup
-    from skglm.penalties import WeightedGroupL2
     from skglm.utils.data import grp_converter
 
 
 class Objective(BaseObjective):
-    name = "Group Lasso objective"
-
-    requirements = ["pip:skglm"]
-
+    name = "Sparse Group Lasso objective"
+
+    requirements = [
+        "numpy'<2'",
```

Comment:
explain why each package is needed for the whole benchmark (not just for a single solver)